gcloud-java-compute/src/test/java/com/google/gcloud/compute/ComputeImplTest.java

```diff
@@ -695,20 +832,355 @@ public void testGetLicenseFromId() {
   }
 
   @Test
-  public void testGetLicenseFromIdWithOptions() {
+  public void testGetGlobalOperation() {
+    EasyMock.expect(
+        computeRpcMock.getGlobalOperation(GLOBAL_OPERATION_ID.operation(), EMPTY_RPC_OPTIONS))
+        .andReturn(globalOperation.toPb());
+    EasyMock.replay(computeRpcMock);
+    compute = options.service();
+    Operation operation = compute.get(GLOBAL_OPERATION_ID);
+    assertEquals(globalOperation, operation);
+  }
+
+  @Test
+  public void testGetGlobalOperationWithSelectedFields() {
     Capture<Map<ComputeRpc.Option, Object>> capturedOptions = Capture.newInstance();
-    LicenseId licenseId = LicenseId.of("project2", "license2");
-    EasyMock.expect(computeRpcMock.getLicense(eq(licenseId.project()), eq(licenseId.license()),
-        capture(capturedOptions)))
-        .andReturn(LICENSE.toPb());
+    EasyMock.expect(computeRpcMock.getGlobalOperation(
+             eq(GLOBAL_OPERATION_ID.operation()), capture(capturedOptions)))
+        .andReturn(globalOperation.toPb());
     EasyMock.replay(computeRpcMock);
     compute = options.service();
-    License license = compute.getLicense(licenseId,  LICENSE_OPTION_FIELDS);
-    assertEquals(LICENSE, license);
-    String selector = (String) capturedOptions.getValue().get(LICENSE_OPTION_FIELDS.rpcOption());
+    Operation operation = compute.get(GLOBAL_OPERATION_ID, OPERATION_OPTION_FIELDS);
+    String selector = (String) capturedOptions.getValue().get(OPERATION_OPTION_FIELDS.rpcOption());
     assertTrue(selector.contains("selfLink"));
-    assertTrue(selector.contains("chargesUseFee"));
-    assertEquals(22, selector.length());
-    assertEquals(LICENSE, license);
+    assertTrue(selector.contains("id"));
+    assertTrue(selector.contains("description"));
+    assertEquals(23, selector.length());
+    assertEquals(globalOperation, operation);
+  }
+
+  @Test
+  public void testListGlobalOperations() {
+    String cursor = "cursor";
+    compute = options.service();
+    ImmutableList<Operation> operationList = ImmutableList.of(globalOperation, globalOperation);
+    Tuple<String, Iterable<com.google.api.services.compute.model.Operation>> result =
+        Tuple.of(cursor, Iterables.transform(operationList,
+            new Function<Operation, com.google.api.services.compute.model.Operation>() {
+              @Override
+              public com.google.api.services.compute.model.Operation apply(Operation operation) {
+                return operation.toPb();
+              }
+            }));
+    EasyMock.expect(computeRpcMock.listGlobalOperations(EMPTY_RPC_OPTIONS)).andReturn(result);
+    EasyMock.replay(computeRpcMock);
+    Page<Operation> page = compute.listGlobalOperations();
+    assertEquals(cursor, page.nextPageCursor());
```

Comment:
This is unit tests for the service, testing next page is out of scope here and won't help to understand if cursor field and other options have been set correctly. This is tested in integration tests, where `iterateAll` is used.